hq/app/views/index.scala.html

```diff
@@ -17,7 +17,7 @@ <h4 class="header light grey-text text-lighten-5">Centralised AWS Account securi
         <div class="row hp-reports">
             <div class="card">
                 <div class="card-content">
-                    <a class="hp__link" href="/iam">
+                    <a class="hp__link" href="@iamReportLink">
```

Comment:
Can possibly hardcode the URL here instead of passing it as a variable to the index controller, but I thought it was neat to have all the dashboard links as vals in the controller. There's only the IAM report link now, but more will be added soon.